tests/integration/common/test_kubeflow_training.py

```diff
@@ -36,22 +36,32 @@ def run_submission_with_function_preprocessor(capsys, deployer="job", builder="a
                                    context_source=gcs_context.GCSContextSource(namespace=namespace))
     else:
         fairing.config.set_builder(builder, base_image=base_image, registry=DOCKER_REGISTRY)
-    fairing.config.set_deployer(deployer, namespace=namespace)
+    fairing.config.set_deployer(deployer, namespace=namespace, cleanup=cleanup)
 
     expected_result = str(uuid.uuid4())
     remote_train = fairing.config.fn(lambda : train_fn(expected_result))
     remote_train()
     captured = capsys.readouterr()
     assert expected_result in captured.out
+    if cleanup:
+        assert "Cleaning up" in caplog.text
```

Comment:
it is better to check if the resources are deleted using k8s api instead of just checking the logs. You can imagine the case where we might print "cleaning up" but the resources may not be cleaned up.

Reply:
@karthikv2k Thanks. Yes, That's better. I'm lazy, haha... Rollbacked last changes, and called k8s python api to check if the tfjob object is removed if set cleanup.